modules/log6x-logging-compatibility-support-matrix.adoc

```diff
@@ -0,0 +1,28 @@
+[id="log6x-logging-compatibility-support-matrix_{context}"]
```

Comment:
missing the following metadata:

// Module included in the following assemblies:
//
// * observability/logging/<remaining_path> (I couldn't find this in the preview links provided and I tried searching this PR diff to see which assembly it might be in but I don't see it in the assemblies here)

:_mod-docs-content-type: REFERENCE

Reply:
I think this was intended to be used in the release notes.